### PR TITLE
[Validator] add MIR card scheme

### PR DESCRIPTION
**Pull request by antonch1989**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no 
| Deprecations? | no 
| Tests pass?   | yes  
| Fixed tickets | #30376 
| License       | MIT

### CODE REVIEW

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Please add a doc PR

**Comment by xabbuh** [comment]
Can you provide a link to a source for the card scheme?

**Comment by antonch1989** [comment]
> Can you provide a link to a source for the card scheme?

https://en.wikipedia.org/wiki/Payment_card_number

File: src/Symfony/Component/Validator/Constraints/CardSchemeValidator.php [hunk 1]
```diff
@@ -78,6 +78,10 @@ class CardSchemeValidator extends ConstraintValidator
             '/^5[1-5][0-9]{14}$/',
             '/^2(22[1-9][0-9]{12}|2[3-9][0-9]{13}|[3-6][0-9]{14}|7[0-1][0-9]{13}|720[0-9]{12})$/',
         ],
+        // Payment system MIR numbers start with 220, then 1 digit from 0 to 4, then 12 digits
+        'MIR' => [
+            '/^220[0-4]{1}[0-9]{12}$/',
```
Comment (xabbuh) [inline src/Symfony/Component/Validator/Constraints/CardSchemeValidator.php]: I don't think the `{1}` part is necessary.

**Comment by antonch1989** [comment]
Docs PR: https://github.com/symfony/symfony-docs/pull/11056

**Review by xabbuh — APPROVED** [review]

**Review by javiereguiluz — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @antonch1989.

**Pull request merged**